blobs/xxx0/README

```diff
@@ -0,0 +1,39 @@
+Coreboot supports generating modified ifd and gbe out of the box.
+To replicate the blobs in this directory (based on coreboot 4.8.1 but simply replace version in paths):
+
+make BOARDS=t400-maximized
+
+This will create the ROM.
+
+Then (considering you git clone heads under ~)
+
+#To generate GBE and IFD
+cd ~/heads/build/coreboot-4.8.1/util/bincfg
+make gen-gbe-ich9m
+make gen-ifd-x200
+mv flashregion_0_fd.bin ../../../../blobs/xxx0/ifd.bin
+mv flashregion_3_gbe.bin ../../../../blobs/xxx0/gbe.bin
+
+#To unlock IFD, permitting to reflash whole flash internally
+cd ~/heads/build/coreboot-4.8.1/util/ifdtool
+make
+cd ~/heads/blobs/xxx0/
+~/heads/build/coreboot-4.8.1/util/ifdtool/ifdtool -u ifd.bin
+mv ifd.bin.new ifd.bin
+
+sha256sum -c hashes.txt
+
+should output:
+gbe.bin: OK
+ifd.bin: OK
+
+IFD contains regions that were outputed to layout.txt through ifdtool -f layout.txt ifd.bin
+It defines a BIOS region of 007fffff – 00003000 = 7FCFFF
```

Comment:
This size is reflected under coreboot config. When boards are not booting (different then beeping), in the past it was because of a size mismatch between ifd defined regions and CBFS_SIZE defined under coreboot config.

Note that first boot after flash might take a couple of seconds more since memory init is done the first time and results need to be injected in CBFS as well.